cpp/ps0001/ps0001_sol.cpp

```diff
@@ -0,0 +1,20 @@
+#include <iostream>
+using namespace std;
+int main()
+{
+    // prompting user for 3 numbers to add 1/2
+    short num1, num2, num3;
+    cout << "please enter the first number" << endl;
+    cin >> num1;
+    cout << "please enter the second number" << endl;
+    cin >> num2;
+    cout << "please enter the third number" << endl;
+    cin >> num3;
+    
+    // calculating & outputting the result 3/3
+    short sum = num1 + num2 + num3;
+    cout << sum;
+    
+    return 0;
+}
+// good work 👍 7/10
```

Comment:
- [ ] program is in the correct path
- [x] no syntax errors
- [x] no bugs occurs while execution
- [x] formatted code to be clear and readable
- [ ] commented some phrases for blocks of code